python_ta/reporters/node_printers.py

```diff
@@ -255,6 +629,28 @@ def render_pep8_errors_e306(msg, _node, source_lines=None):
     yield from render_context(msg.line, msg.line + 2, source_lines)
 
 
+def render_pep8_errors_e714(msg, _node, source_lines=None):
```

Comment:
This algorithm doesn't generalize to other examples. Remove it from this PR, we'll leave it for future work.